kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/confirmation/ConfirmationPaneCommonViewModel.java

```diff
@@ -0,0 +1,80 @@
+package dev.ikm.komet.kview.mvvm.viewmodel.confirmation;
+
+import dev.ikm.komet.framework.events.Evt;
+import dev.ikm.komet.framework.events.EvtBusFactory;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import org.carlfx.cognitive.viewmodel.SimpleViewModel;
+
+import java.util.UUID;
+
+/**
+ * The view model for the confirmation pane.  This view model manages the text that is displayed
+ * on the confirmation pane, and the notification topic and event that is sent when the close
+ * button is pressed.
+ */
+public class ConfirmationPaneCommonViewModel extends SimpleViewModel {
+
+    // properties
+    public static final String NOTIFICATION_TOPIC = "notificationTopic";
+    public static final String NOTIFICATION_EVENT = "notificationEvent";
+
+    private StringProperty title = new SimpleStringProperty();
+    private StringProperty message = new SimpleStringProperty();
+    private ConfirmationMessages confirmationMessage;
```

Comment:
Can we create properties for title and message in the constructor?

Tip: Create a public enum or a separate file of the enum properties names?